example/spec/helpers/Providers.js

```diff
@@ -100,10 +101,36 @@ const throwIfApiReturnFail = (apiResult) => {
   }
 };
 
+const providerExists = async (stackName, id) => {
+  let response;
+  const exists = await pRetry(
+    async () => {
+      try {
+        response = await providersApi.getProvider({
```

Comment:
Did you intend to wrap this retry in the invokeApi retry loop?   

If not, given the default invokeApi defaults for the inner pretry loop 

```
      retries: 3,
      maxTimeout: 10000,
```

and it retrying on anything other than the expected result (200), this might need tuned a bit to avoid unneded timeouts/retries.

Reply:
(This applies to the other instances of the pattern in this PR.

Reply:
[`bdf616f` (#2533)](https://github.com/nasa/cumulus/pull/2533/commits/bdf616fa6e7ab7056a80c11244041f74f9424b2b)